src/constants/chain-info.ts

```diff
@@ -13,6 +13,8 @@ export interface ChainInfoObject {
   };
 }
 
+export const AvailableBlockChains = ["ETH", "MATIC"];
+
```

Comment:
so for polygon and polygon mumbai we need to use `MATIC` chain value? just curious what happens when we use the wrong one

Reply:
it will throw an error stating error in document, this check only happens for document that has the auto network switching. so if u have a polygon mumbai document as of now that doesn't have the auto networking switching, it will skip this check and proceed, given that the user selected the correct network from the dropdown.